ExaFunctions/Utilities/QueryUtilities.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Text;
+
+namespace Exasol
+{
+    public class QueryUtilities
+    {
+        public static JObject RunQuery(string connString, string query)
+        {
+            JObject result = new JObject();
+
+            //https://docs.telerik.com/reporting/knowledge-base/how-to-register-db-provider-factory-in-net-core-project
+            //C:\Windows\Microsoft.NET\Framework64\v4.0.30319\machine.Config -> on installation an entry gets written here + the dll gets added to the GAC
+            //in .NET core there's no GAC (and no machine.config)
+            //DbProviderFactories.RegisterFactory("Exasol.EXADataProvider", "Exasol.EXADataProvider");
+            DbProviderFactories.RegisterFactory("Exasol.EXADataProvider", "Exasol.EXADataProvider.EXAProviderFactory, EXADataProvider, Version=5.0.0.0, Culture=neutral, PublicKeyToken=ec874333d1454516");
+            DbProviderFactory factory = null;
+            //try
+            //{
+            factory = DbProviderFactories.GetFactory("Exasol.EXADataProvider");
+            Console.WriteLine("Found Exasol driver");
+
+            //TODO: use a using here
```

Comment:
use a using?
Please create a GitHub ticket for this TODO

Reply:
I already made the changes :) No need for a TODO then.